esx_service/cli/vmdkops_admin.py

```diff
@@ -388,8 +389,11 @@ def generate_ls_rows():
         policy = get_policy(metadata, path)
         size_info = get_vmdk_size_info(path)
         created, created_by = get_creation_info(metadata)
+        access = get_access(metadata)
```

Comment:
i think the names should be get_access_mode() and get_attach_mode() ? this way it will consistent with the column names , now it's a bit confusing.